stage1/enter_kvm/enter_kvm.go

```diff
@@ -140,7 +141,7 @@ func init() {
 	flag.StringVar(&podPid, "pid", "", "podPID")
 	flag.StringVar(&appName, "appname", "", "application to use")
 
-	log = rktlog.New(os.Stderr, "kvm", false)
+	log, diag, _ = rktlog.NewLogSet("kvm", false)
```

Comment:
TODO for later (added in #1131):

If `rktlog.NewLogSet` and `exec.LookPath("ssh")` are moved in `main()`, it will not need to call `SetDebug()` manually.